test/test_collection.py

```diff
@@ -213,5 +213,19 @@ def test_cli_tool(self):
         self.assertNotEqual(res.stdout, "")
         self.assertTrue(res.returncode == 0)
 
+    def test_imports(self):
+        try:
+            from pymongoexplain import ExplainCollection
+        except:
+            print("Could not import!")
+            assert False
+        try:
```

Comment:
Suggest leaving a newline between the two sub-tests to make this more readable.

Reply:
Done.